satellite/satellitedb/testdata/postgres.v53.sql

```diff
@@ -0,0 +1,347 @@
+-- AUTOGENERATED BY gopkg.in/spacemonkeygo/dbx.v1
+-- DO NOT EDIT
+CREATE TABLE accounting_rollups (
+    id bigserial NOT NULL,
+    node_id bytea NOT NULL,
+    start_time timestamp with time zone NOT NULL,
+    put_total bigint NOT NULL,
+    get_total bigint NOT NULL,
+    get_audit_total bigint NOT NULL,
+    get_repair_total bigint NOT NULL,
+    put_repair_total bigint NOT NULL,
+    at_rest_total double precision NOT NULL,
+    PRIMARY KEY ( id )
+);
+CREATE TABLE accounting_timestamps (
+    name text NOT NULL,
+    value timestamp with time zone NOT NULL,
+    PRIMARY KEY ( name )
+);
+CREATE TABLE bucket_bandwidth_rollups (
+    bucket_name bytea NOT NULL,
+    project_id bytea NOT NULL,
+    interval_start timestamp NOT NULL,
+    interval_seconds integer NOT NULL,
+    action integer NOT NULL,
+    inline bigint NOT NULL,
+    allocated bigint NOT NULL,
+    settled bigint NOT NULL,
+    PRIMARY KEY ( bucket_name, project_id, interval_start, action )
+);
+CREATE TABLE bucket_storage_tallies (
+    bucket_name bytea NOT NULL,
+    project_id bytea NOT NULL,
+    interval_start timestamp NOT NULL,
+    inline bigint NOT NULL,
+    remote bigint NOT NULL,
+    remote_segments_count integer NOT NULL,
+    inline_segments_count integer NOT NULL,
+    object_count integer NOT NULL,
+    metadata_size bigint NOT NULL,
+    PRIMARY KEY ( bucket_name, project_id, interval_start )
+);
+CREATE TABLE bucket_usages (
+    id bytea NOT NULL,
+    bucket_id bytea NOT NULL,
+    rollup_end_time timestamp with time zone NOT NULL,
+    remote_stored_data bigint NOT NULL,
+    inline_stored_data bigint NOT NULL,
+    remote_segments integer NOT NULL,
+    inline_segments integer NOT NULL,
+    objects integer NOT NULL,
+    metadata_size bigint NOT NULL,
+    repair_egress bigint NOT NULL,
+    get_egress bigint NOT NULL,
+    audit_egress bigint NOT NULL,
+    PRIMARY KEY ( id )
+);
+CREATE TABLE injuredsegments (
+    path bytea NOT NULL,
+    data bytea NOT NULL,
+    attempted timestamp,
+    PRIMARY KEY ( path )
+);
+CREATE TABLE irreparabledbs (
+    segmentpath bytea NOT NULL,
+    segmentdetail bytea NOT NULL,
+    pieces_lost_count bigint NOT NULL,
+    seg_damaged_unix_sec bigint NOT NULL,
+    repair_attempt_count bigint NOT NULL,
+    PRIMARY KEY ( segmentpath )
+);
+CREATE TABLE nodes (
+    id bytea NOT NULL,
+    address text NOT NULL,
+    last_net text NOT NULL,
+    protocol integer NOT NULL,
+    type integer NOT NULL,
+    email text NOT NULL,
+    wallet text NOT NULL,
+    free_bandwidth bigint NOT NULL,
+    free_disk bigint NOT NULL,
+    piece_count bigint NOT NULL,
```

Comment:
I don't know if this file should match the current default database schema if that's the case then it this column need to have the default

Reply:
I think dbx is responsible for setting the default with the `autoinsert`  attribute.

Reply:
Yess, that's right, this is more a question if these test schemas should match 100% the implementation DB schema.